Dockerfile.dapper

```diff
@@ -1,21 +1,16 @@
-FROM golang:1.16
-RUN apt-get update && \
-    apt-get install -y xz-utils zip rsync
+FROM registry.suse.com/bci/golang:1.19
 
-RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.46.2;
+RUN GO_VERSION=$(echo "$GOLANG_VERSION" | sed -r 's/^([[:digit:]]+.[[:digit:]]+).*/\1/') && \
+    zypper -n install docker go${GO_VERSION}-race
 
+ENV GOLANGCI_LINT v1.53.3
```

Comment:
Will renovate bot be able to identify this is a version as an ENV? In Webhook it puts the version on the `curl` command and renovate bot is able to update this automatically:
https://github.com/rancher/webhook/pull/251/files

Reply:
[Not at the moment](https://github.com/rancher/renovate-config/blob/5d08c09fac6c3f2d27474fb6996bc6a0766776d9/default.json#L83-L93), but we probably should update the renovate config to accommodate that.